sdk/src/time_stamp.rs

```diff
@@ -631,7 +631,21 @@ pub(crate) fn verify_timestamp(ts: &[u8], data: &[u8]) -> Result<TstInfo> {
                     validator.validate(&sig_val.to_bytes(), &tbs, &signing_key_der)
                 }
 
-                #[cfg(not(feature = "openssl"))]
```

Comment:
Can we still make a regular build without openssl? This needs to work with the c2pa-min repo.

Reply:
Yes. The build logic for native builds without openssl remains unchanged. I've only added a new section for synchronous calls to `verify_timestamp` for wasm32